databases/week2/lesson2_homework_part3.sql

```diff
@@ -0,0 +1,30 @@
+-- You should do these queries on the database hyf_lesson2, which we created last class.
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+USE hyf_lesson2;
+
+SELECT *
+FROM user
+WHERE email LIKE '%@spotify.com';
```

Comment:
You are supposed to get task assigned to a user having an email consists %spotify...
```
SELECT user.name,user.email, task.title,task.id
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user ON user.id = user_task.user_id
WHERE user.email LIKE '%@spotify.com';
```

Reply:
I got a little confused here. Getting better with some practice.